src/main/scala/nl.knaw.dans.easy.bag2deposit/DepositPropertiesFactory.scala

```diff
@@ -29,45 +29,47 @@ case class DepositPropertiesFactory(configuration: Configuration, idType: IdType
   def create(bagInfo: BagInfo, ddm: Elem): Try[PropertiesConfiguration] = Try {
     val ddmIds: NodeSeq = ddm \ "dcmiMetadata" \ "identifier"
 
-    def getBaseUrn(versionOf: UUID) = configuration.bagIndex.getURN(versionOf).unsafeGetOrThrow
+    def getBaseUrn(versionOf: UUID) = {
+      bagSource match {
+        case VAULT => configuration.bagIndex.getURN(versionOf).unsafeGetOrThrow
+        case FEDORA => bagInfo.baseUrn.getOrElse(throw InvalidBagException("version-of without base-urn"))
+      }
+    }
 
     def getIdType(idType: String) = ddmIds
       .find(_.hasType(s"id-type:$idType"))
       .getOrElse(throw InvalidBagException(s"no $idType"))
       .text
 
+    val doi = getIdType("DOI")
+    val urn = getIdType("URN")
     val fedoraId = ddmIds
       .find(_.text.startsWith("easy-dataset"))
       .getOrElse(throw InvalidBagException("no fedoraID"))
       .text
 
-    val doi = getIdType("DOI")
-    val urn = getIdType("URN")
-    val src = bagSource match {
-      case BagSource.VAULT => "the vault"
-      case BagSource.FEDORA => "EASY-fedora"
-    }
-
     new PropertiesConfiguration() {
       addProperty("state.label", "SUBMITTED")
-      addProperty("state.description", s"This deposit was extracted from $src and is ready for processing")
-      addProperty("deposit.origin", bagSource)
+      addProperty("state.description", submittedStateDescription(bagSource))
+      addProperty("deposit.origin", bagSource.toString)
       addProperty("creation.timestamp", bagInfo.created)
       addProperty("depositor.userId", bagInfo.userId)
-      addProperty("bag-store.bag-id", bagInfo.uuid)
       addProperty("bag-store.bag-name", bagInfo.bagName)
       addProperty("identifier.doi", doi)
       addProperty("identifier.urn", urn)
       addProperty("identifier.fedora", fedoraId)
+      if (bagSource == VAULT) {
+        addProperty("bag-store.bag-id", bagInfo.uuid)
+        addProperty("dataverse.sword-token", bagInfo.versionOf.getOrElse(bagInfo.uuid))
+      }
       addProperty("dataverse.bag-id", "urn:uuid:" + bagInfo.uuid)
```

Comment:
`dataverse.bag-id` and bag-store.bag-name are only expected when `bagSource == VAULT`